chefs_soils.py

```diff
@@ -29,396 +29,12 @@
-            _hv_dic['legalLandDescription'] = helper.create_untitled_municipal_land_desc(
-                _hvs,
-                helper.chefs_hv_param('untitledMunicipalLand'),
-                helper.chefs_hv_param('untitledMunicipalLandDesc'))
-
-        _hv_dic['crownLandFileNumbers'] = helper.create_land_file_numbers(_hvs, helper.chefs_hv_param('crownLandFileNumbers'))
-        _hv_dic['receivingSiteLandUse'] = helper.create_receiving_site_lan_uses(_hvs, helper.chefs_hv_param('receivingSiteLandUse'))
-        _hv_dic['hvsConfirmation'] = _hvs.get(helper.chefs_hv_param('hvsConfirmation'))
-        _hv_dic['dateSiteBecameHighVolume'] = helper.convert_simple_datetime_format_in_str(_hvs.get(helper.chefs_hv_param('dateSiteBecameHighVolume')))
-        _hv_dic['howRelocatedSoilWillBeUsed'] = _hvs.get(helper.chefs_hv_param('howRelocatedSoilWillBeUsed'))
-        _hv_dic['soilDepositIsALR'] = _hvs.get(helper.chefs_hv_param('soilDepositIsALR'))
-        _hv_dic['soilDepositIsReserveLands'] = _hvs.get(helper.chefs_hv_param('soilDepositIsReserveLands'))
-        _hv_dic['qualifiedProfessionalFirstName'] = _hvs.get(helper.chefs_hv_param('qualifiedProfessionalFirstName'))
-        _hv_dic['qualifiedProfessionalLastName'] = _hvs.get(helper.chefs_hv_param('qualifiedProfessionalLastName'))
-        _hv_dic['qualifiedProfessionalType'] = _hvs.get(helper.chefs_hv_param('qualifiedProfessionalType'))
-        _hv_dic['qualifiedProfessionalOrganization'] = _hvs.get(helper.chefs_hv_param('qualifiedProfessionalOrganization'))
-        _hv_dic['professionalLicenceRegistration'] = _hvs.get(helper.chefs_hv_param('professionalLicenceRegistration'))
-        _hv_dic['qualifiedProfessionalAddress'] = _hvs.get(helper.chefs_hv_param('qualifiedProfessionalAddress'))
-        _hv_dic['qualifiedProfessionalCity'] = _hvs.get(helper.chefs_hv_param('qualifiedProfessionalCity'))
-        _hv_dic['qualifiedProfessionalProvince'] = _hvs.get(helper.chefs_hv_param('qualifiedProfessionalProvince'))
-        _hv_dic['qualifiedProfessionalCountry'] = _hvs.get(helper.chefs_hv_param('qualifiedProfessionalCountry'))
-        _hv_dic['qualifiedProfessionalPostalCode'] = _hvs.get(helper.chefs_hv_param('qualifiedProfessionalPostalCode'))
-        _hv_dic['qualifiedProfessionalPhoneNumber'] = _hvs.get(helper.chefs_hv_param('qualifiedProfessionalPhoneNumber'))
-        _hv_dic['qualifiedProfessionalEmail'] = _hvs.get(helper.chefs_hv_param('qualifiedProfessionalEmail'))
-        _hv_dic['signaturerFirstAndLastName'] = _hvs.get(helper.chefs_hv_param('signaturerFirstAndLastName'))
-        _hv_dic['dateSigned'] = helper.convert_simple_datetime_format_in_str(_hvs.get(helper.chefs_hv_param('dateSigned')))
-        _hv_dic['createAt'] = helper.get_create_date(_hvs, helper.chefs_hv_param('form'), helper.chefs_hv_param('createdAt'))
-        _hv_dic['confirmationId'] = _confirmation_id
-    return _hv_dic
+
+logging.basicConfig(level=LOGLEVEL)
```

Comment:
I feel it's better to add a format to the basic config along with level which would print the timestamp and the message for each log entrry. something like below.
FORMAT = '%(asctime)s %(message)s'
logging.basicConfig(format=FORMAT)